src/test/sandbox/TestSandbox.wsb

```diff
@@ -0,0 +1,17 @@
+<Configuration>
+    <MappedFolders>
+        <MappedFolder>
+            <HostFolder>..\..\..\build</HostFolder>
+            <SandboxFolder>C:\build</SandboxFolder>
+            <ReadOnly>true</ReadOnly>
```

Comment:
Perhaps this needs to change with the logging setup to write dumps to C:\build..
Or should there perhaps be another mapping that's writable from sandbox back to host for logs\dumps?